### DISCUSSION TITLE
[HELP]run modbus demo error

### DISCUSSION THREAD

**Discussion by leonJiang1992** [eclipse-kura/kura#5282] [General]
log detail like this :
![image](https://github.com/eclipse/kura/assets/172479559/d0c464d0-eecd-46ca-b0fe-894cb3b28a1a)

**Comment by MMaiero** [comment]
Hello,
sorry for the delay in the response but somewhat we didn't get a notification.
The error seems to be related to a missing dependency on the org.eclipse.kura.message bundle. It may be that the dependency is not properly declared or not present in the target environment you are using.
Maybe detailing your setup can help in understanding your issue. 
Best regards,
Matteo
